lib/rubocop/cop/style/swap_values.rb

```diff
@@ -79,26 +79,15 @@ def replacement(x_assign)
         end
 
         def lhs(node)
-          case node.type
-          when :casgn
-            namespace, name, = *node
-            if namespace
-              "#{namespace.const_name}::#{name}"
-            else
-              name.to_s
-            end
+          if node.casgn_type? && node.namespace
+            "#{node.namespace.const_name}::#{node.name}"
           else
-            node.children[0].to_s
+            node.name.to_s
           end
```

Comment:
Now that rubocop/rubocop-ast#330 is released in v1.34.1, you could do this instead:

```suggestion
          if node.casgn_type?
            "#{'::' if node.absolute?}#{node.const_name}"
          else
            node.name.to_s
          end
```